ambari-server/src/main/java/org/apache/ambari/server/topology/Blueprint.java

```diff
@@ -74,7 +76,59 @@
   Setting getSetting();
 
   /**
+<<<<<<< HEAD
    * @return the set of stack (mpack) IDs associated with the blueprint
+=======
```

Comment:
Merge conflict.

Reply:
Thanks.  Two further minor issues:

1. The first `@return` (the one about mpacks) is not appropriate for `getServices()`.
2. Can you please also remove
    ```
    >>>>>>> trunk
    ```
    from line 129?  It does not show up as compile error, because it happens to be in a javadoc comment.